bindings/wasm/src/did/wasm_document.rs

```diff
@@ -497,3 +660,57 @@ extern "C" {
   #[wasm_bindgen(typescript_type = "DIDUrl | string")]
   pub type UDIDUrlQuery;
 }
+
+#[wasm_bindgen]
+extern "C" {
+  #[wasm_bindgen(typescript_type = "DID[]")]
+  pub type DIDArray;
+}
+
+#[wasm_bindgen]
+extern "C" {
+  #[wasm_bindgen(typescript_type = "Service[]")]
+  pub type ArrayService;
+}
+
+#[wasm_bindgen]
+extern "C" {
+  #[wasm_bindgen(typescript_type = "VerificationMethod[]")]
+  pub type ArrayMethods;
+}
+
+#[wasm_bindgen]
+extern "C" {
+  #[wasm_bindgen(typescript_type = "Map<string, any>")]
+  pub type JSMap;
+}
```

Comment:
Nit: name suggestion, either `MapStringAny` or `WasmProperties`. I prefer the former for its verboseness when looking at the return type.

```suggestion
#[wasm_bindgen]
extern "C" {
  #[wasm_bindgen(typescript_type = "Map<string, any>")]
  pub type MapStringAny;
}
```